src/foremast/utils/dns.py

```diff
@@ -91,6 +91,40 @@ def update_dns_zone_record(env, zone_id, **kwargs):
 
     LOG.debug('Route53 JSON Response: \n%s', pformat(response))
 
+def delete_existing_cname(env, zone_id, dns_name):
+    """Function to delete an existing CNAME record. This is
+    used when updating to multi-region for deleting old records.
+    The record can not just be upserted since it changes types.
+
+    Args:
+        env (str): Deployment environment.
+        zone_id (str): Route53 zone id.
+        dns_name (str): FQDN of application's dns entry to add/update.
+    """
+    startrecord = None
+    newrecord_name = dns_name
+    client = boto3.Session(profile_name=env).client('route53')
+    response = client.list_resource_record_sets(
+        HostedZoneId=zone_id
+    )
+    for item in response['ResourceRecordSets']:
+        if item['Name'][:-1] == newrecord_name and item['Type'] == 'CNAME':
```

Comment:
Can you do `old_record_name = item['name'].rstrip('.')`? That would be clearer to me.